### PR TITLE
MON-3110: Add servicemesh_member_count metric to telemetry for OpenShift Service Mesh

### PR DESCRIPTION
**Pull request by bmangoen**
* [ ] I added CHANGELOG entry for this change.
* [ X] No user facing changes, so no entry in CHANGELOG was needed.

Add Service Mesh Control Plane Namespaces counter to telemetry (cf. [MON-3110](https://issues.redhat.com/browse/MON-3110))

### CODE REVIEW

File: Documentation/data-collection.md [hunk 1]
```diff
@@ -939,6 +939,11 @@ data:
     #
     # cluster:vsphere_infrastructure_failure_domains:max shows how many vSphere failure domains, vCenters, datacenters and datastores are configured in a cluster.
     # '{__name__="cluster:vsphere_infrastructure_failure_domains:max"}'
+    #
+    # owners: (https://github.com/maistra/istio-operator, OpenShift Service Mesh team)
+    #
+    # cluster:servicemesh_member_count:sum represents the Service Mesh Control Plane Namespaces counter sum of the cluster
+    - '{__name__="cluster:servicemesh_member_count:sum"}'
```
Comment (simonpasquier) [inline Documentation/data-collection.md:946]: can you link the code where the metric is generated?

**Comment by simonpasquier** [comment]
/hold

depends on https://github.com/maistra/istio-operator/pull/1119

**Comment by jan--f** [comment]
/remove-lifecycle stale
This looks good to me. I left a comment in the respective PR adding the metric though.

**Pull request closed without merging**